app/src/androidTest/java/com/android/unio/ui/authentication/AccountDetailsTest.kt

```diff
@@ -70,105 +72,128 @@ class AccountDetailsTest {
 
   @Test
   fun testEverythingIsDisplayed() {
-    composeTestRule.onNodeWithTag("AccountDetailsTitleText").assertIsDisplayed()
-    composeTestRule
-        .onNodeWithTag("AccountDetailsTitleText")
-        .assertTextEquals("Tell us about yourself")
-    composeTestRule.onNodeWithTag("AccountDetailsFirstNameTextField").assertIsDisplayed()
-    composeTestRule
-        .onNodeWithTag("AccountDetailsFirstNameText", useUnmergedTree = true)
-        .assertTextEquals("First name")
-    composeTestRule.onNodeWithTag("AccountDetailsLastNameTextField").assertIsDisplayed()
-    composeTestRule
-        .onNodeWithTag("AccountDetailsLastNameText", useUnmergedTree = true)
-        .assertTextEquals("Last name")
-    composeTestRule.onNodeWithTag("AccountDetailsBioTextField").assertExists()
-    composeTestRule
-        .onNodeWithTag("AccountDetailsBioText", useUnmergedTree = true)
-        .assertTextEquals("Bio")
-    composeTestRule.onNodeWithTag("AccountDetailsProfilePictureText").assertExists()
-    composeTestRule
-        .onNodeWithTag("AccountDetailsProfilePictureText")
-        .assertTextEquals("Maybe add a profile picture?")
-    composeTestRule.onNodeWithTag("AccountDetailsProfilePictureIcon").assertExists()
-    composeTestRule.onNodeWithTag("AccountDetailsInterestsButton").assertExists()
-    composeTestRule.onNodeWithTag("AccountDetailsSocialsButton").assertExists()
-    composeTestRule.onNodeWithTag("AccountDetailsContinueButton").assertExists()
+    composeTestRule.onNodeWithTag(AccountDetailsTestTags.TITLE_TEXT).assertIsDisplayed()
+    composeTestRule
+        .onNodeWithTag(AccountDetailsTestTags.FIRST_NAME_TEXT, useUnmergedTree = true)
+        .assertIsDisplayed()
+    composeTestRule
+        .onNodeWithTag(AccountDetailsTestTags.LAST_NAME_TEXT, useUnmergedTree = true)
+        .assertIsDisplayed()
+    composeTestRule.onNodeWithTag(AccountDetailsTestTags.BIOGRAPHY_TEXT_FIELD).assertExists()
+    composeTestRule.onNodeWithTag(AccountDetailsTestTags.PROFILE_PICTURE_TEXT).assertExists()
+    composeTestRule.onNodeWithTag(AccountDetailsTestTags.PROFILE_PICTURE_ICON).assertExists()
+    composeTestRule.onNodeWithTag(AccountDetailsTestTags.INTERESTS_BUTTON).assertExists()
+    composeTestRule.onNodeWithTag(AccountDetailsTestTags.SOCIALS_BUTTON).assertExists()
+    composeTestRule.onNodeWithTag(AccountDetailsTestTags.CONTINUE_BUTTON).assertExists()
   }
 
   @Test
   fun testOutLinedTextFieldsWorkCorrectly() {
-    composeTestRule.onNodeWithTag("AccountDetailsFirstNameTextField").performTextInput("John")
-    composeTestRule.onNodeWithTag("AccountDetailsLastNameTextField").performTextInput("Doe")
-    composeTestRule.onNodeWithTag("AccountDetailsBioTextField").performTextInput("I am a student")
+    composeTestRule
+        .onNodeWithTag(AccountDetailsTestTags.FIRST_NAME_TEXT_FIELD)
+        .performTextInput("John")
```

Comment:
It could be great to store the user info in a variable to avoid string replication. These ones are reused in line 104 anbd below